ttnn/cpp/ttnn/operations/data_movement/concat/concat_pybind.hpp

```diff
@@ -52,14 +53,16 @@ Keyword Args:
                 const int dim,
                 std::optional<ttnn::Tensor> &optional_output_tensor,
                 std::optional<ttnn::MemoryConfig>& memory_config,
+                const int groups,
                 uint8_t queue_id) {
-                    return self(queue_id, tensors, dim, memory_config, optional_output_tensor);
```

Comment:
Can you add the constraints to the doc string for the pybind. Like the fact that group concat is only for two tensors and only support height sharding.